src/style/_information.scss

```diff
@@ -7,32 +7,29 @@
         margin: 0 0 20px 0;
         a
         {
-            font-size: 12px;
+            @include regular_text($color: $info-link-font-color);
         }
     }
-    pre 
+    pre
     {
         font-size: 14px;
     }
     p, li, table
     {
-        font-size: 14px;
-
-        @include text_body();
+        @include regular_text($size: 1.4em);
     }
 
     h1, h2, h3, h4, h5
     {
         @include text_body();
     }
 
-    a
+    div > a, a.info__extdocs.link
```

Comment:
I think this might be a bit too ambiguous - maybe add the classname to be more specific:
`div.opblock-tag > a`

Reply:
there are multiple a tags under the div thats why its like that

Reply:
But you are styling all direct descendant `a` elements in all `div` elements...
Adding the div class `.opblock-tag` would be a bit more specific